ClientApp/src/app/components/miscellaneous/sprint-dropdown/sprint-dropdown.component.ts

```diff
@@ -32,23 +35,74 @@ export class SprintDropdownComponent implements OnInit {
 
   sprintStarted!: boolean;
 
+  sprintFormattedStartDate: string | undefined = '';
+  sprintFormattedEndDate: string | undefined = '';
+
   collapsed!: boolean;
   hidden!: boolean;
 
   optionsMenuItems: MenuItem[] = [];
 
+  manageIncompleteTasksDialogVisible = false;
+  incompleteTasksAction: 'moveToBacklog' | 'moveToSprint' | 'markAsDone' = 'moveToBacklog';
+  incompleteTasksProcess: 'sprintCompletion' | 'sprintDeletion' = 'sprintCompletion';
+
+  availableSprints: SprintData[] = [];
+  selectedAvailableSprint!: SprintData;
+  availableSprintErrorVisibility: 'hidden' | 'visible' = 'hidden';
+
+  availableBacklogs: SprintData[] = [];
+  selectedAvailableBacklog!: SprintData;
+  availableBacklogErrorVisibility: 'hidden' | 'visible' = 'hidden';
+
+  numIncompleteTasks = 0;
+  numCompleteTasks = 0;
+
+  incompletePoints = 0;
+  completePoints = 0;
+
+  sprintReportDialogVisible = false;
+
+  sprintReportTasksData!: any;
+  sprintReportPointsData!: any;
+
+  green500 = getComputedStyle(document.documentElement).getPropertyValue('--green-500');
```

Comment:
Just curious, what does this do over just passing in hex values?

Reply:
that green value comes from the primeng theme, so if we ever decided to have custom on the fly themes, the values would change. now that it's this late in the term though, theming is more of a creature comfort for sure, low priority in my opinion. in general I've been trying to use the variable colors provided by primeng wherever possible instead of hard coded colors